src/functionalTest/resources/scenarios/RIA-2976-aip-send-notifications-request-more-time-appellant-case-officer-clarifying-questions.json

```diff
@@ -68,32 +69,18 @@
           }
         ],
         "notificationsSent": [
-          {
-            "id": "1020_SUBMIT_TIME_EXTENSION_CASE_OFFICER",
-            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
-          },
           {
             "id": "1020_MAKE_AN_APPLICATION_APPELLANT_AIP_EMAIL",
             "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
           },
           {
-            "id": "1020_MAKE_AN_APPLICATION_APPELLANT_AIP_SMS",
```

Comment:
@burale is there any reason we're moving the AIP SMS notification from this test?

Reply:
Added back